Formula/j/julia.rb

```diff
@@ -146,7 +144,7 @@ def install
 
     # Remove library versions from MbedTLS_jll, nghttp2_jll and others
     # https://git.archlinux.org/svntogit/community.git/tree/trunk/julia-hardcoded-libs.patch?h=packages/julia
-    %w[MbedTLS nghttp2 LibGit2 OpenLibm].each do |dep|
+    %w[MbedTLS nghttp2 LibGit2 OpenLibm SuiteSparse].each do |dep|
       (buildpath/"stdlib").glob("**/#{dep}_jll.jl") do |jll|
         inreplace jll, %r{@rpath/lib(\w+)(\.\d+)*\.dylib}, "@rpath/lib\\1.dylib"
         inreplace jll, /lib(\w+)\.so(\.\d+)*/, "lib\\1.so"
```

Comment:
Let's just open these files once:
```suggestion
    %w[MbedTLS nghttp2 LibGit2 OpenLibm SuiteSparse].each do |dep|
      inreplace (buildpath/"stdlib").glob("**/#{dep}_jll.jl") do |s|
        s.gsub!(%r{@rpath/lib(\w+)(\.\d+)*\.dylib}, "@rpath/lib\\1.dylib")
        s.gsub!(/lib(\w+)\.so(\.\d+)*/, "lib\\1.so")
```